src/server/models/userModel.ts

```diff
@@ -7,7 +7,7 @@ import {GitHub} from '../../utils/github';
 interface LoginDetails {
   username: string;
   token: string;
-  scopes: string[];
+  scopes: string[]|null;
```

Comment:
Can't we just keep it as is? [] can the empty list.

Reply:
The reason for this is so that we can differentiate between - we know the scopes and there are none granted and when we simply don't know the scopes.

This is likely to change when we move to Firestore, so for now I'd like to keep this as is.